static/src/javascripts/projects/commercial/modules/third-party-tags/krux.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import config from 'lib/config';
+import { getCookie } from 'lib/cookies';
+import { local } from 'lib/storage';
+
+const kruxUrl: string = '//cdn.krxd.net/controltag?confid=JVZiE3vn';
+
+const retrieve = (n: string): string => {
+    const k: string = `kx${n}`;
+
+    return local.getRaw(k) || getCookie(`${k}=([^;]*)`) || '';
+};
+
+const getKruxSegments = (): Array<string> =>
+    retrieve('segs') ? retrieve('segs').split(',') : [];
+
+const shouldRun = config.switches.krux;
+
+export { shouldRun, kruxUrl, getKruxSegments };
```

Comment:
The issue is that `kruxUrl` isn't what should be exported. It needs to be exported as `url` like the [original](https://github.com/guardian/frontend/pull/16955/files#diff-a9822ad8d10ea82b93073afa80b8e6c9L24)